Composer/packages/server/src/externalContentProvider/powerVirtualAgentsProvider.ts

```diff
@@ -0,0 +1,173 @@
+import { createWriteStream } from 'fs';
+import { ensureDirSync, remove } from 'fs-extra';
+import fetch, { RequestInit } from 'node-fetch';
+import { join } from 'path';
+
+import logger from '../logger';
+import { authService } from '../services/auth/auth';
+
+import { BotContentInfo, ContentProviderMetadata, ExternalContentProvider } from './externalContentProvider';
+
+const log = logger.extend('pva-provider');
+
+const COMPOSER_1P_APP_ID = 'ce48853e-0605-4f77-8746-d70ac63cc6bc';
```

Comment:
Just checking if it's fine to have this checked in with the code

Reply:
Yup, it's not a secret 👍